tf2_eigen/package.xml

```diff
@@ -1,20 +1,26 @@
 <?xml version="1.0"?>
```

Comment:
I think that we should stay in sync with upstream regardless of the status of the opened PRs. Then it's up to the PR opener to rebase their branches accordingly if they happen to conflict.